google/cloud/storage/internal/curl_download_request.cc

```diff
@@ -28,6 +28,20 @@ namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace internal {
 
+extern "C" std::size_t CurlDownloadRequestWrite(char* ptr, size_t size,
```

Comment:
Nit: It is a little strange that a function callable from C is marked as returning a C++ standard type.

Reply:
Fixed.